app/jobs/participant_outcomes/send_to_tra_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ParticipantOutcomes
+  class SendToTraJob < ApplicationJob
```

Comment:
since we're using the following names in the db: `qualified_teachers_api` [see](https://github.com/DFE-Digital/early-careers-framework/pull/2979/files#diff-3230112ce8ebd07001ef3181045fc6ce9b66791bb35135078abf06d574a776d1R5)  maybe we can rename this job to `SendToQualifiedTeachersAPI`?